baseframe/Makefile

```diff
@@ -19,13 +20,26 @@ tinymce:
 	rm -rf $(tinymce4_zip) tinymce
 
 codemirror:
-	curl -L -O http://codemirror.net/$(codemirror_zip)
+	curl -L -O https://github.com/codemirror/CodeMirror/archive/$(codemirror4_zip)
 	unzip $(codemirror_zip)
+	# Remove old unversioned codemirror
 	rm -rf static/js/codemirror
-	mkdir static/js/codemirror
-	mv codemirror-*/addon static/js/codemirror
-	mv codemirror-*/keymap static/js/codemirror
-	mv codemirror-*/lib static/js/codemirror
-	mv codemirror-*/mode static/js/codemirror
-	mv codemirror-*/theme static/js/codemirror
-	rm -rf $(codemirror_zip) codemirror-*
+	mkdir static/js/codemirror4
+	mv codemirror-*/addon static/js/codemirror4
+	mv codemirror-*/keymap static/js/codemirror4
+	mv codemirror-*/lib static/js/codemirror4
+	mv codemirror-*/mode static/js/codemirror4
+	mv codemirror-*/theme static/js/codemirror4
+	rm -rf $(codemirror4_zip) codemirror4-*
+
+	curl -L -O https://github.com/codemirror/CodeMirror/archive/$(codemirror5_zip)
+	unzip $(codemirror_zip)
+	mkdir static/js/codemirror5
```

Comment:
Put these in the `codemirror` folder, not `codemirror5` (unless backward incompatible).